db/dcrpg/internal/txstmts.go

```diff
@@ -131,7 +131,7 @@ const (
 	SelectStakeTxByHash   = `SELECT id, block_hash, block_index FROM transactions WHERE tx_hash = $1 and tree=1;`
 
 	IndexTransactionTableOnBlockIn = `CREATE UNIQUE INDEX uix_tx_block_in
-		ON transactions(block_hash, block_index, tree);`  // with cockroach: STORING (tx_hash, block_hash)
+		ON transactions(block_hash, block_index, tree);` // with cockroach: STORING (tx_hash, block_hash)
```

Comment:
Let's remove the comment.  `gofmt` keeps switching it around and we're not working on cockroachdb yet.

Reply:
ok. Noted